explainer.md

```diff
@@ -364,19 +348,7 @@ This code allows a customized set of properties to each new Realm - e.g. `docume
 
 In principle, the Realm proposal does not provide the controls for the module graphs. Every new Realm initializes its own module graph, while any invocation to `Realm.prototype.import()` method, or by using `import()` when evaluating code inside the realm, will populate this module graph. This is analogous to same-domain iframes, and VM in nodejs.
 
-However, the [Compartments]() proposal plans to provide the low level hooks to control the module graph per Realm. This is one of the intersection semantics between the two proposals. Although, the biggest challenge when sharing modules across realms is the hazard of the identity discontinuity. For example, when interacting with a module evaluated in another Realm:
```

Comment:
I don't think this adds much since this proposal is not attempting to solve this problem. as for the identity discontinuity, we have a section dedicated to that... not needed here